gradio/layouts.py

```diff
@@ -323,15 +323,16 @@ def __init__(self, *, scale: int = 0, min_width: int = 0, **kwargs):
             scale: relative width compared to adjacent Columns. For example, if Column A has scale=2, and Column B has scale=1, A will be twice as wide as B.
             min_width: minimum pixel width of Column, will wrap if not sufficient screen space to satisfy this value. If a certain scale value results in a column narrower than min_width, the min_width parameter will be respected first.
         """
-        self.scale = scale
-        self.min_width = min_width
+        self.scale = int(scale)
```

Comment:
why is this casting necessary? the constructor expects an int

Reply:
The type annotation doesn't prevent anyone from passing in something else.

This makes e.g. an inadvertent `max_width=None` (which would be valid for some other components) fail at construction time instead of when something is attempting to do math with it.

Reply:
I don't think we should guard against people passing the wrong type in, do we do that anywhere else?

Reply:
I see, `assert isinstance(min_width, int)` would be clearer in intent

Reply:
@aliabid94 `assert`s could be compiled out in `-O` mode. See https://beta.ruff.rs/docs/rules/assert/

Reply:
@pngwn Why not? I think it makes sense here since the overwhelmingly more common types for `scale` and `min_width` are

```
scale: int | None = None,
min_width: int | None = None,
```

but this component only accepts `int`s and will break otherwise.

Reply:
Yeah I think it's a good idea to check here, even if we don't do it generally.